scripts/automation/docker/deploy-hawkular.sh

```diff
@@ -88,17 +88,23 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-${OC_} get projects | grep ${OS_PROJECT_}
+${OC_} get projects | grep "${OS_PROJECT_}[[:space:]]"
 if [ $? -eq 0 ]; then
     echo "Deleting project:${OS_PROJECT_}"
-    ${OC_} delete project ${OS_PROJECT_}
-    if [ $? -ne 0 ]; then
+    OC_RESULT=`${OC_} delete project ${OS_PROJECT_}`
+    if [ $? -ne 0 ] ; then
         echo "Failed to delete project: ${OS_PROJECT_}"
-        exit 1
+        echo ${OC_RESULT} | grep --quiet "The system is ensuring all content is removed from this namespace"
```

Comment:
In case the project is already terminating (g.e. because of previous execution of this script) this will skip the 'exit 1'